ark-testing/tests/aspd.rs

```diff
@@ -28,7 +28,7 @@ async fn check_aspd_version() {
 
 #[tokio::test]
 async fn fund_asp() {
-	let context = TestContext::generate();
+	let context = TestContext::new("aspd/fund_aspd");
```

Comment:
These things are unfortunate, but I guess nothing we can do about them.

Reply:
Agreed, I wish I could find a better way to tackle this. But I just don't think it's worth it